### DISCUSSION TITLE
Before moving to the 🇺🇸, where did Albert Einstein live?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#742] [Galaxy Trivia] [answered]
**Bonus:** What state did he move to?

**Comment by SANKARAMDAS** [comment, marked as answer]
German,
 112 Mercer Street in Princeton, Mercer County, New Jersey, United States

**Comment by peterbaccaro** [comment]
Before moving to the United States, Albert Einstein lived in **Germany**, his country of birth. He was born in Ulm, Germany in 1879 and later lived in Munich and Berlin.

Albert Einstein moved to the state of **New Jersey** in the United States. Specifically, he settled in the town of Princeton, which is located in Mercer County, New Jersey. Princeton is known for being home to Princeton University, which has a prestigious faculty and student body in the fields of science, mathematics, and engineering. Einstein worked at the Institute for Advanced Study, which is also located in Princeton, and spent the rest of his life there until his death in 1955.
